src/components/search.js

```diff
@@ -39,11 +40,20 @@ export default class Search extends Component {
         render={data => {
           return (
             <div>
-              <input
-                type="text"
+              <TextField
+                className={'searchField'}
+                style={{color:'gray',backgroundColor:'white'}}
```

Comment:
How does this work together with the mui theme? Shouldn't it use the theme colors here?

Reply:
Also to get the round corners, the variant outlined should be used

Reply:
![Screenshot from 2019-07-08 12-01-25](https://user-images.githubusercontent.com/1449049/60801936-275c0400-a178-11e9-834e-40db5bb83acf.png)

It would be nice if the icon is grey as well.

Reply:
Yes we can have a gray icon too no worries

Reply:
Yes it is the mui theme colors but we overdrive them with different colors,

I'll add round corners for sure

Reply:
@x5engine I don't think that it is good to overwrite elements like this. Can't we use a light version of the theme?

Reply:
yes sure we can use a different version but the other proper way to override colors and styles with the classes

like this

```
const useStyles = makeStyles(theme => ({
  margin: {
    margin: theme.spacing(1),
  },
}));
```

Reply:
@x5engine that looks good!